content/manuals/desktop/use-desktop/kubernetes.md

```diff
@@ -1,48 +1,46 @@
 ---
 description: See how you can deploy to Kubernetes on Docker Desktop
 keywords: deploy, kubernetes, kubectl, orchestration, Docker Desktop
-title: Deploy on Kubernetes with Docker Desktop
+title: Explore the Kubernetes view
 linkTitle: Deploy on Kubernetes
 aliases:
 - /docker-for-windows/kubernetes/
 - /docker-for-mac/kubernetes/
 - /desktop/kubernetes/
-weight: 60
+- /desktop/features/kubernetes/
+weight: 50
 ---
 
 Docker Desktop includes a standalone Kubernetes server and client, as well as Docker CLI integration, enabling local Kubernetes development and testing directly on your machine.
 
-The Kubernetes server runs as a single or multi-node cluster, within Docker container(s). This lightweight setup helps you explore Kubernetes features, test workloads, and work with container orchestration in parallel with other Docker functionalities.
+The Kubernetes server runs as a single or multi-node cluster, within Docker containers. This lightweight setup helps you explore Kubernetes features, test workloads, and work with container orchestration in parallel with other Docker functionality such as Swarm services or standalone containers.
```

Comment:
I'd probably avoid calling out swarm services/stacks since they're not very well-supported.

```suggestion
The Kubernetes server runs as a single or multi-node cluster, within Docker containers. This lightweight setup helps you explore Kubernetes features, test workloads, and work with container orchestration in parallel with other Docker features.
```